tensorflow/core/common_runtime/mkl_layout_pass.cc

```diff
@@ -1406,9 +1406,11 @@ class MklLayoutRewritePass : public GraphOptimizationPass {
 
           Tensor tensor;
           if (!tensor.FromProto(*proto)) {
-            TF_CHECK_OK(errors::InvalidArgument(
-                "Could not construct Tensor from TensorProto in node: ",
-                node->name()));
+            TF_CHECK_OK(Status(
+                absl::StatusCode::kInvalidArgument,
+                absl::StrCat(
+                    "Could not construct Tensor from TensorProto in node: ",
+                    node->name())));
```

Comment:
Why can't we use `absl::InvalidArgumentError` like I had suggested? I think it's more compact. Example:
```suggestion
            TF_CHECK_OK(
                absl::InvalidArgumentError(
                    absl::StrCat(
                        "Could not construct Tensor from TensorProto in node: ",
                        node->name())));
```

Reply:
Agree, fixed it.